docs/reference/operators.rst

```diff
@@ -154,3 +154,51 @@ The midpoint of the corresponding edge of the Object’s bounding box, inheritin
 (top | bottom) (front | back) (left | right) of *Object*
 --------------------------------------------------------
 The corresponding corner of the Object’s bounding box, inheriting the Object's orientation.
+
+.. _temporal operators:
+
+Temporal Operators
+=======================
+
+Temporal operators can be used inside require statements to constrain how a dynamic scenario evolves over time.
+The semantics of these operators are taken from RV-LTL [B10]_ to properly model the finite length of Scenic simulations.
+
+.. _always {condition}:
+.. _always:
+
+always *condition*
+------------------
+Require the given condition to hold throughout the execution of the dynamic scenario.
+
+.. _eventually {condition}:
+.. _eventually:
+
+eventually *condition*
+----------------------
+Require the given condition to hold at some point during the execution of the dynamic scenario.
+
+.. _next {condition}:
+.. _next:
+
+next *condition*
+----------------
+Require the given condition to hold at the next time step of the dynamic scenario.
+
+.. _{condition} until {condition}:
+.. _until:
+
+*condition* until *condition*
```

Comment:
Is there a name for LHS and RHS of the until operator, like "hypothesis" and "conclusion" for `implies`?

Reply:
The closest I can think of would be precondition/postcondition, but this doesn't seem to exactly meet their definitions in the contexts I've heard them used.

Reply:
I don't know of specific names for these -- what you have is clear, I think.